### DISCUSSION TITLE
Sample Script for Aerostructural Optimisation

### DISCUSSION THREAD

**Discussion by AbhijithMoni** [mdolab/AerostructuralOptBenchmark#2] [Q&A] [answered]
Does the Repository include an example/sample Python code that demonstrates how to execute the aerostructural optimization outlined in the benchmark model?

This would be helpful for users to understand and implement the optimization process.

Thanks

**Comment by A-CGray** [comment, marked as answer]
Hi @AbhijithMoni 

I might add scripts for some parts of the model setup, but I think it's unlikely that I'll post a full ready-to-go optimization script as that is not really the purpose of this repo. The purpose is mainly to provide a model for other people to use with their own tools.

**Reply by AbhijithMoni** [reply]
Thanks for your response @A-CGray.
